src/Simulation/Simulators/QuantumSimulator/Qubit.cs

```diff
@@ -20,10 +20,16 @@ public QSimQubit(int id, uint simulatorId) : base(id)
                 this.SimulatorId = simulatorId;
             }
 
+            public QSimQubit(int id, uint simulatorId, string label) : this(id, simulatorId) {
```

Comment:
This one also looks like it should be a separate change

Reply:
Reverted as well, same mistake